clients/python/llmengine/model.py

```diff
@@ -366,3 +368,51 @@ def delete(cls, model: str) -> DeleteLLMEndpointResponse:
         """
         response = cls._delete(f"v1/llm/model-endpoints/{model}", timeout=DEFAULT_TIMEOUT)
         return DeleteLLMEndpointResponse.parse_obj(response)
+
+    @classmethod
+    def download(
+        cls,
+        model_name: str,
+        download_format: str = "huggingface",
```

Comment:
It seems like we've used `hugging_face` elsewhere in LLM Engine. I think the motivation for this was due to Hugging Face being two words?
![image](https://github.com/scaleapi/llm-engine/assets/2041983/025283d2-e166-4b8e-80b6-bff64f9c96a0)

Reply:
oh okay, that's a relatively easy fix (currently we're not actually doing anything with the format token anyway)